GitUp/Application/AppDelegate.m

```diff
@@ -613,6 +620,10 @@ - (IBAction)cloneRepository:(id)sender {
   }
 }
 
+- (IBAction)cloneRepository:(id)sender {
+  [self _cloneRepositoryFromURLString:@""];
```

Comment:
Oh, I thought your intent was to immediately prompt where to clone and clone, not show the repo URL dialog. I guess it makes sense like this.